src/main/java/hudson/plugins/promoted_builds/conditions/ManualCondition.java

```diff
@@ -95,18 +98,28 @@ public ParameterDefinition getParameterDefinition(String name) {
         return null;
     }
 
-    public Set<String> getUsersAsSet() {
+    public Set<String> getUsersAsSet(AbstractBuild<?,?> build) {
         if (users == null || users.equals(""))
             return Collections.emptySet();
 
         Set<String> set = new HashSet<String>();
         for (String user : users.split(",")) {
             user = user.trim();
 
-            if (user.trim().length() > 0)
+            LOGGER.log(Level.FINER, "ManualCondition.getUsersAsSet(): user={0}", user);
+            if (user.startsWith("$")){
+                user = user.substring(1);
+                if (user.startsWith("{") && user.endsWith("}"))
```

Comment:
I would prefer to allow either `$NAME` or `${NAME}`, but not both. What happens if there is a user named `$NAME`? I don't think users can have `$` in their name after a recent security fix, but I am not 100% sure.

Reply:
the reason I allow both $NAME and ${NAME} because
Email-ext plugin and token-macro plugin support these 2 syntax.

But if you insist, 
I could allow only one of the case,
and which form do you prefert to support?

Reply:
If email-ext and token-macro support both, then it probably makes sense to support both here as well. I'd leave it the way you have it for now.